generator/final_processor_country.cpp

```diff
@@ -309,7 +309,6 @@ void CountryFinalProcessor::AddFakeNodes()
 
 void CountryFinalProcessor::DropProhibitedSpeedCameras()
 {
-  static auto const speedCameraType = classif().GetTypeByPath({"highway", "speed_camera"});
```

Comment:
Better to remove static here, rather than introduce static var inside ForEach

Reply:
Why is it better?

Does it make sense to initialize speedCameraType as a member variable to save some processing time?

Reply:
I can't say 100%, but pretty sure that static var is always checking some kind of sync primitive. So you introduced N primitive checks putting static into ForEach(Feature).

DropProhibitedSpeedCameras called once.

Reply:
> Better to remove static here

I have removed the static.

> I can't say 100%, but pretty sure that static var is always checking some kind of sync primitive. 

Disassembly showed that, on top of a bunch of copies of the variable, another bunch of guard variables were also generated by the compiler, with the name `<guard variable for [...] const::speedCameraType>`